### ISSUE TITLE
Open fsi results in error

### ISSUE THREAD

**Issue by 7sharp9** [fsprojects/atom-fsharp#64]
```
'/Library/Frameworks/Mono.framework/Versions/4.0.1/lib/mono/4.5/fsharpi' could not be spawned. Is it installed and on your path? If so please open an issue on the package spawning the process.
```
And in the console:
```
Uncaught Error: spawn /Library/Frameworks/Mono.framework/Versions/4.0.1/lib/mono/4.5/fsharpi ENOENT
```
not only is it in my path but the path above is fully resolved anyway?

**Comment by adamchester** [comment]
Same error here.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `0.5.3`

**Comment by evilpilaf** [comment]
Gettting the same problem, installed today the latest version and after opening Atom I get this message
